Android/app/src/main/java/app/intra/socks/OverrideSocksHandler.java

```diff
@@ -166,6 +168,8 @@ public void doConnect(Session session, CommandMessage commandMessage) throws IOE
       // - VALUE : total transfer over the lifetime of a socket
       // - PORT : TCP port number (i.e. protocol type)
       // - DURATION: socket lifetime in seconds
+      // - LATENCY: TCP handshake latency in milliseconds
+      // - DELTARTT - Difference between data latency and TCP handshake latency, in milliseconds.
```

Comment:
Why do you save the delta and not the data latency? That would be clearer.

Reply:
Maybe call it FIRST_BYTE_MS?

Reply:
OK, switched to FIRST_BYTE_MS.  We can compute the difference during the analysis.